server/gokbg3/src/integration-test/groovy/gokbg3/rest/IdentifierTestSpec.groovy

```diff
@@ -44,6 +44,7 @@ class IdentifierTestSpec extends AbstractAuthSpec {
       accept('application/json')
     }
     then:
+//    resp.status == 200 // WRONG!
```

Comment:
Can be removed?

Reply:
sometimes, the test returns a 200 status for no obvious reason. I suspect some html session to be alive still in the server for being responsible to validate the user.